components/menu/icons.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Icons
+page_title: Menu for Blazor | Icon
+description: Icons and images in the Menu for Blazor
+slug: menu-icons
+tags: telerik,blazor,menu,icon,iconclass,image
+published: True
+position: 2
+---
+
+# Menu Icons
+
+You can put an image, icon class or a font icon for each item in the menu to illustrate its purpose for your end users. To apply them, use the following properties:
+
+* for a font icon, populate the `IconField` parameter of the component or provide an `Icon` property in the data model.
```

Comment:
fix this up in the same manner as the Drawer article - including factological explanations here, indentations and field names that clearly denote what the fields hold.

Reply:
Done with fb6a044a0f7b443962247314e76ac0a28fe459fc